tests/instrumentation-framework/src/main/java/org/hawkular/apm/tests/dockerized/ProjectVersion.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2015-2016 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.hawkular.apm.tests.dockerized;
+
+import java.io.IOException;
+import java.util.Properties;
+
+/**
+ * Class for accessing version of the application at runtime.
+ *
+ * @author Pavol Loffay
+ */
+public class ProjectVersion {
+
+    private static final Properties properties;
+
+    static {
+        properties = new Properties();
+        try {
+            properties.load(ProjectVersion.class.getClassLoader().getResourceAsStream("project.properties"));
```

Comment:
I like this approach, but why not read from the MANIFEST.MF, like the Status endpoint does? Reference: https://github.com/hawkular/hawkular-commons/blob/master/hawkular-rest-status/src/main/java/org/hawkular/commons/rest/status/ManifestUtil.java